genapkovl-lima.sh

```diff
@@ -73,6 +73,17 @@ rc_add networking default
 
 rc_add sshd default
 
+rc_add local default
+
+mkdir -p "${tmp}/etc/local.d/"
+makefile root:root 0755 "$tmp/etc/local.d/lima.start" << EOF
+sed -i 's/#UsePAM no/UsePAM yes/g' /etc/ssh/sshd_config
```

Comment:
Doing this on _every_ start (as opposed to the first time) seems like it might annoy people who were modifying this intentionally. But I guess the install happens every time too…